Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/EditorVertexSelection.h

```diff
@@ -313,18 +323,24 @@ namespace AzToolsFramework
     //! EditorVertexSelectionVariable provides selection and editing for a variable length number of
     //! vertices. New vertices can be inserted/added or removed from the collection.
     template<typename Vertex>
-    class EditorVertexSelectionVariable : public EditorVertexSelectionBase<Vertex>
+    class EditorVertexSelectionVariable
+        : public EditorVertexSelectionBase<Vertex>
+        , private AzToolsFramework::EditorVertexSelectionVariableRequestBus::Handler
     {
     public:
         AZ_CLASS_ALLOCATOR_DECL
 
-        EditorVertexSelectionVariable() = default;
-        EditorVertexSelectionVariable(EditorVertexSelectionVariable&&) = default;
```

Comment:
update: Please can we add these back

Reply:
Removing these actually broke tests, so I had to restore them. Thanks for pointing it out :D

Reply:
Nice one thanks!